src/internal/packager/deploy.go

```diff
@@ -52,7 +52,7 @@ func Deploy() {
 
 	// Load the config from the extracted archive zarf.yaml
 	spinner.Updatef("Loading the zarf package config")
-	if err := config.LoadConfig(tempPath.base + "/zarf.yaml"); err != nil {
+	if err := config.LoadConfig(tempPath.base+"/zarf.yaml", true); err != nil {
```

Comment:
Can we do `filepath.Join()` here instead of concatenating strings. It'll make our lives easier once we start supporting the Windows binary.